docs/verify/__use_estimator.mdx

```diff
@@ -0,0 +1,248 @@
+
+  ```python
+  from qiskit.primitives import BackendEstimator
+  from <some_qiskit_provider> import QiskitProvider
+
+  provider = QiskitProvider()
+  backend = provider.get_backend('backend_name')
+  estimator = BackendEstimator(backend)
+  ```
+
+  There are some providers that implement primitives natively (see [this page](https://qiskit.org/providers/#primitives) for more details).
+</Admonition>
+
+## Initialize observables
+
+The first step is to define the observables whose expected value you want to compute. Each observable can be any `BaseOperator`, like the operators from [`qiskit.quantum_info`](../api/qiskit/quantum_info).
+Among them it is preferable to use [`qiskit.quantum_info.SparsePauliOp`](../api/qiskit/qiskit.quantum_info.SparsePauliOp).
+
+```python
+from qiskit.quantum_info import SparsePauliOp
+
+observable = SparsePauliOp(["II", "XX", "YY", "ZZ"], coeffs=[1, 1, -1, 1])
+```
+
+## Initialize quantum circuit
+
+Then you need to create the [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit)s for which you want to obtain the expected value.
+
+```python
+from qiskit import QuantumCircuit
+
+qc = QuantumCircuit(2)
+qc.h(0)
+qc.cx(0,1)
+qc.draw("mpl")
+```
+
+![Initial QuantumCircuit](/images/verify/use_estimator/initialize.png)
+
+<Admonition type="note" title="Note">
+  The [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit) you pass to [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) must not include any measurements.
+</Admonition>
+
+## Initialize the `Estimator`
+
+Then, you need to instantiate an [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator).
+
+```python
+from qiskit.primitives import Estimator
+
+estimator = Estimator()
+```
+
+## Run and get results
+
+Now that you have defined your `estimator`, you can run your estimation by calling the [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run) method,
+which returns an instance of [`qiskit.providers.JobV1`](../api/qiskit/qiskit.providers.JobV1). You can get the results from the job (as a [`qiskit.primitives.EstimatorResult`](../api/qiskit/qiskit.primitives.EstimatorResult) object)
+with the [`qiskit.providers.JobV1.result`](../api/qiskit/qiskit.providers.JobV1#result) method.
+
+```python
+job = estimator.run(qc, observable)
+result = job.result()
+print(result)
+```
+
+```python
+EstimatorResult(values=array([4.]), metadata=[{}])
+```
+
+While this example only uses one [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit) and one observable, if you want to get expectation values for multiple circuits and observables you can
+pass a `list` of [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit)s and a list of `BaseOperator`s to the [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run) method. Both `list`s must have
```

Comment:
The Sphinx had bad escaping of this `list` with the \` character. I removed the extra character.